webui/react/src/pages/TrialDetails/useTrialMetrics.ts

```diff
@@ -112,7 +112,9 @@ export const useTrialMetrics = (trials: (TrialDetails | undefined)[]): TrialMetr
 
   const loadableMetrics = useMetricNames(experimentIds, handleMetricNamesError);
   const metricNamesLoaded = Loadable.isLoaded(loadableMetrics);
-  const metrics = Loadable.getOrElse([], loadableMetrics);
+  const metrics = useMemo(() => {
+    return Loadable.getOrElse([], loadableMetrics);
```

Comment:
When `Loadable.getOrElse` returns empty array, it triggers `maximum update warnings`